tests/phpunit/tests/dependencies.php

```diff
@@ -148,4 +148,92 @@ public function test_enqueue_before_register() {
 
 		$this->assertContains( 'one', $dep->queue );
 	}
+
+	/**
+	 * Data provider for test_get_etag.
+	 *
+	 * @return array
+	 */
+	public function data_provider_get_etag() {
```

Comment:
```suggestion
	/**
	 * Data provider for test_get_etag_scripts and test_get_etag_styles.
	 *
	 * @return array[]
	 */
	public function data_provider_get_etag() {
```

Reply:
Actually the right annotation is @return array<string,array<string,array<string,string>|string>>

Reply:
For data providers core tends to use the simple `array[]` as they may change as the test is updated and can be inferred by the docblock on the actual tests.

Reply:
Ok, I'll update the annotation.